app/pods/publications/publication/controller.js

```diff
@@ -27,51 +38,58 @@ export default class PublicationController extends Controller {
       .toUpperCase();
   }
 
+  // TODO get status
+  // TODO set selected item from the status value
+
   get getPublicationNumberSidePanel() {
-    console.log('P number', this.model.get('publicationNumber'));
     if (this.model.get('publicationNumber')) {
       this.publicationNumber = this.model.get('publicationNumber');
     }
     return this.publicationNumber;
   }
 
+  get currentDate() {
+    return moment();
+  }
+
   get getTranslationDate() {
-    console.log('P translationDate', this.model.get('translateBefore'));
     if (this.model.get('translateBefore')) {
       const date = this.model.get('translateBefore');
-      this.translationDate = moment(date).format('DD-MM-YYYY');
+      this.translationDate = moment(date)
+        .format('DD-MM-YYYY');
     }
+    this.expiredTranslationDate = this.checkDate(this.translationDate);
     return this.translationDate;
   }
 
   get getPublicationBeforeDate() {
-    console.log('P beforeDate', this.model.get('publishBefore'));
     if (this.model.get('publishBefore')) {
-      const date  = this.model.get('publishBefore');
-      this.publicationBeforeDate = moment(date).format('DD-MM-YYYY');
+      const date = this.model.get('publishBefore');
+      this.publicationBeforeDate = moment(date)
+        .format('DD-MM-YYYY');
     }
+    this.expiredPublicationBeforeDate = this.checkDate(this.publicationBeforeDate);
```

Comment:
Als je een publicatie opent en je een datum zet, dan wordt die datum globaal opgeslagen via de `this.publicationBeforeDate` en `this.expiredPublicationBeforeDate` ... als je dan een nieuwe publicatie maakt en daar is de property `publishBefore` automatisch null, dan zal uw getter de value teruggeven van de vorige keer dat je een datum ingevuld hebt omdat de variabele nooit gereset wordt.